client/src/components/AuthForm/AuthForm.js

```diff
@@ -51,27 +56,26 @@ const AuthForm = (props) => {
           />
         </figure>
 
-        {
-          inputs.map(({type}) => renderInputField({type, key: type}))
-        }
+        {inputs.map(({ type }) => renderInputField({ type, key: type }))}
```

Comment:
Key is type, pretty weird, you can add the index..

Reply:
Added index here